lib/_stream_transform.js

```diff
@@ -217,10 +218,10 @@ function done(stream, er, data) {
   var ts = stream._transformState;
 
   if (ws.length)
-    throw new Error('Calling transform done when ws.length != 0');
+    throw new errors.Error('ERR_CALLING_WS_LENGTH');
 
   if (ts.transforming)
-    throw new Error('Calling transform done when still transforming');
+    throw new errors.Error('ERR_CALLING_IN_TRANSFORMING');
```

Comment:
Maybe to be consistent with my other suggestion:
'ERR_TRANSFORM_ALREADY_TRANSFORMING' String is already specific to transform so making that part of the ID makes sense to me.